apps/api/test/report-request-util.ts

```diff
@@ -29,7 +37,9 @@ export const getHAs = async () => {
 };
 
 // add new applicant
-export const addApplicant = async (applicant: IENApplicantCreateUpdateDTO) => {
+export const addApplicant = async (
+  applicant: IENApplicantCreateUpdateDTO,
+): Promise<ApplicantRO> => {
```

Comment:
I pulled the PR and was getting errors when running the "api-integration-test" command. 
![image](https://user-images.githubusercontent.com/100959887/222772635-6a91fedd-27cf-4cf3-891f-b09b1b789814.png)
Seems like giving this generic caused some issues for 00-applicant file.

Reply:
Getting same errors for report 5 as well.
![image](https://user-images.githubusercontent.com/100959887/222775224-a4cad653-34c9-4f18-a034-3994d9905569.png)